integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXScaleTestParameters.java

```diff
@@ -81,4 +99,24 @@ public String getIrrelevantHostGroup() {
     public void setIrrelevantHostGroup(String irrelevantHostGroup) {
         this.irrelevantHostGroup = irrelevantHostGroup;
     }
+
+    public AdjustmentType getAdjustmentType() {
+        return adjustmentType;
+    }
+
+    public void setAdjustmentType(String adjustmentTypeName) {
+        adjustmentType = Arrays.stream(AdjustmentType.values())
```

Comment:
It's a bit complicated, why not just Enum.valueOf(AdjustmentType.class, adjustmentTypeName)?